Projekty/Projekt1/3.1_Kraszewski_Niewiadowski/EDA.ipynb

```diff
@@ -0,0 +1,2242 @@
+{
```

Comment:
<p>Przydałby się jakiś komentarz – są braki (ile procentowo?), ile jest kolumn z chociaż jednym nullem, czy to jest problem, co można z tym zrobić. Trochę opowiedzieliście o tym podczas prezentacji, więc nie ucinam za to punktów, ale warto żeby w notebooku znalazł się krótki opis. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>